feature.log

```diff
@@ -0,0 +1,30 @@
+benchmarking Bench.Database.LSMTree.Internal.Lookup/prepLookups for a single run/defaulty, ? actual pages/Bloomfilter query
```

Comment:
did you meant to commit this?

Reply:
Yes, it's only a temporary commit to communicate the benchmark results I'm getting